### PR TITLE
BXMSDOC-37917.2 RN- 7.2.1 RN update

### PR DESCRIPTION
**Pull request by emmurphy1**
Added Fixed in 7.2.1 chapter

[RHDM draft](http://file.ork.redhat.com/~emmurphy/BXMSDOC-3791-dm/#rn-721-fixed-issues-ref)
[RHPAM draft](http://file.ork.redhat.com/~emmurphy/BXMSDOC-3791-pam/#rn-721-fixed-issues-ref)

### CODE REVIEW

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Cannot set a sub-process process from different project as reusable [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1667]]
+* Cannot attach boundary events to service tasks in the new process designer [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1773]]
+
+== {KIE_SERVER}
+
+* {KIE_SERVER} loses connection to {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1718[RHPAM-1718]]
+
+== OpenShift
+
+* Add https support to Openshift images [https://issues.jboss.org/browse/RHPAM-1814[RHPAM-1814]]
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Add https support to smart router for OpenShift images?

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+== Upgrade tool
+
+* Add support for Red Had  kie-server and controller in update tool. [https://issues.jboss.org/browse/RHPAM-1824[RHPAM-1824]]
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Red Hat ;)

**Review by krisv — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
+[id='rn-721-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.2.1
+
+{PRODUCT} 7.2.1 provides increased stability and fixed issues listed in this section.
+
+
+
+== {CENTRAL}
+ifdef::DM[]
+* The BRL condition `is contained in the (comma separated) list` does not produce a multi-select drop down in the guided decision table or guided rule template[https://issues.jboss.org/browse/RHDM-483[RHDM-483]]
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]:
```suggestion
* The BRL condition `is contained in the (comma separated) list` does not produce a multi-select drop down in the guided decision table or guided rule template [https://issues.jboss.org/browse/RHDM-483[RHDM-483]]
```

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+endif::PAM[]
+
+== Decision engine
+* An error occurs when using the 7.1.x version of the `kie-maven-plugin` in JBDS [https://issues.jboss.org/browse/RHDM-802[RHDM-802]]
+ifdef::DM[]
+* Incremental compilation fails when calling static method in RHS [https://issues.jboss.org/browse/RHDM-709[RHDM-709]]
+endif::DM[]
+ifdef::PAM[]
+* Incremental compilation fails when calling static method in RHS[https://issues.jboss.org/browse/RHPAM-1453[RHPAM-1453]]
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]:
```suggestion
* Incremental compilation fails when calling static method in RHS [https://issues.jboss.org/browse/RHPAM-1453[RHPAM-1453]]
```

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+* Variables in the task description are no longer resolved after migration [https://issues.jboss.org/browse/RHPAM-1688[RHPAM-1688]]
+* A process does not stop after an exception occurs when the `AsyncMode` environment entry value is `true` [https://issues.jboss.org/browse/RHPAM-1760[RHPAM-1760]]
+
+== Process designer
+* Cannot set a sub-process process from different project as reusable [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1773]]
+* Cannot attach boundary events to service tasks in the new process designer [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1773]]
+* Issues related to boundary events in the new process designer [https://issues.jboss.org/browse/RHPAM-1809[RHPAM-1809]]
+
+== {KIE_SERVER}
+* {KIE_SERVER} loses connection to {CENTRAL}[https://issues.jboss.org/browse/RHPAM-1718[RHPAM-1718]]
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]:
```suggestion
* {KIE_SERVER} loses connection to {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1718[RHPAM-1718]]
```

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+== Maven repository
+* The RESTeasy version in the Maven repository does not match the version in the binaries [https://issues.jboss.org/browse/RHPAM-1845[RHPAM-1845]]
+* Missing JWS dependencies in offline Maven repository distribution [https://issues.jboss.org/browse/RHPAM-1715[RHPAM-1715]]
+
+== OpenShift
+* Add https support to Smart Router OpenShift images[https://issues.jboss.org/browse/RHPAM-1814[RHPAM-1814]]
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]:
```suggestion
* Add https support to Smart Router OpenShift images [https://issues.jboss.org/browse/RHPAM-1814[RHPAM-1814]]
```

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+* Incremental compilation fails when calling static method in RHS[https://issues.jboss.org/browse/RHPAM-1453[RHPAM-1453]]
+
+== Process engine
+
+* Variables in the task description are no longer resolved after migration [https://issues.jboss.org/browse/RHPAM-1688[RHPAM-1688]]
+* A process does not stop after an exception occurs when the `AsyncMode` environment entry value is `true` [https://issues.jboss.org/browse/RHPAM-1760[RHPAM-1760]]
+
+== Process designer
+* Cannot set a sub-process process from different project as reusable [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1773]]
+* Cannot attach boundary events to service tasks in the new process designer [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1773]]
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]:
```suggestion
* Cannot attach boundary events to service tasks in the new process designer [https://issues.jboss.org/browse/RHPAM-1773[RHPAM-1773]]
```

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+ifdef::PAM[]
+* Incremental compilation fails when calling static method in RHS[https://issues.jboss.org/browse/RHPAM-1453[RHPAM-1453]]
+
+== Process engine
+
+* Variables in the task description are no longer resolved after migration [https://issues.jboss.org/browse/RHPAM-1688[RHPAM-1688]]
+* A process does not stop after an exception occurs when the `AsyncMode` environment entry value is `true` [https://issues.jboss.org/browse/RHPAM-1760[RHPAM-1760]]
+
+== Process designer
+* Cannot set a sub-process process from different project as reusable [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1773]]
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]:
```suggestion
* Cannot set a sub-process process from different project as reusable [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1667]]
```

Comment (tomasdavidorg) [reply doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Incorrect name of the link.

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc (hunk 7 above)
Comment (tomasdavidorg) [reply doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Incorrect link.

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc (hunk 6 above)
Comment (tomasdavidorg) [reply doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Missing space character.

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc (hunk 5 above)
Comment (tomasdavidorg) [reply doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Missing space character.

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc (hunk 4 above)
Comment (tomasdavidorg) [reply doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Missing space character.

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc (hunk 3 above)
Comment (tomasdavidorg) [reply doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: Missing space character.

**Review by tomasdavidorg — APPROVED** [review]

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 9]
```diff
@@ -0,0 +1,51 @@
+[id='rn-721-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.2.1
+
+{PRODUCT} 7.2.1 provides increased stability and fixed issues listed in this section.
+
+
+
+== {CENTRAL}
+ifdef::DM[]
+* The BRL condition `is contained in the (comma separated) list` does not produce a multi-select drop down in the guided decision table or guided rule template [https://issues.jboss.org/browse/RHDM-483[RHDM-483]]
```
Comment (sterobin) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: "drop-down list" here and below (per IBM)

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+== Decision engine
+* An error occurs when using the 7.1.x version of the `kie-maven-plugin` in JBDS [https://issues.jboss.org/browse/RHDM-802[RHDM-802]]
+ifdef::DM[]
+* Incremental compilation fails when calling a static method in RHS [https://issues.jboss.org/browse/RHDM-709[RHDM-709]]
+endif::DM[]
+ifdef::PAM[]
+* Incremental compilation fails when calling a static method in RHS [https://issues.jboss.org/browse/RHPAM-1453[RHPAM-1453]]
+
+== Process designer
+* Cannot set a sub-process process from different project as reusable [https://issues.jboss.org/browse/RHPAM-1667[RHPAM-1667]]
```
Comment (sterobin) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: "Cannot set a sub-process from a different project as reusable"

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+== {KIE_SERVER}
+* {KIE_SERVER} loses connection to {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1718[RHPAM-1718]]
+* Variables in the task description are no longer resolved after migration [https://issues.jboss.org/browse/RHPAM-1688[RHPAM-1688]]
+* A process does not stop after an exception occurs when the `AsyncMode` environment entry value is `true` [https://issues.jboss.org/browse/RHPAM-1760[RHPAM-1760]]
+
+== Maven repository
+* Missing JWS dependencies in offline Maven repository distribution [https://issues.jboss.org/browse/RHPAM-1715[RHPAM-1715]]
+
+== OpenShift
+* Add https support to Smart Router OpenShift images [https://issues.jboss.org/browse/RHPAM-1814[RHPAM-1814]]
```
Comment (sterobin) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: "Add HTTPS support"

File: doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+== Update tool
+* Add support for {KIE_SERVER} and the controller in the update tool [https://issues.jboss.org/browse/RHPAM-1824[RHPAM-1824]]
```
Comment (sterobin) [inline doc-content/enterprise-only/release-notes/rn-721-fixed-issues-ref.adoc]: "and the {CONTROLLER} in"

**Review by sterobin — COMMENTED** [review]
@emmurphy1 done. Minor tweaks. All links are good.

**Pull request closed without merging**